docs/guides/how_to_guides/creating_batches/how_to_load_a_pandas_dataframe_as_a_batch.rst

```diff
@@ -84,106 +84,174 @@ This guide will help you load a Pandas DataFrame as a Batch for use in creating
+                class_name: PandasExecutionEngine
+              module_name: great_expectations.datasource
+              class_name: Datasource
+              data_connectors:
+                my_runtime_data_connector:
+                  class_name: RuntimeDataConnector
+                  batch_identifiers:
+                    - some_key_maybe_pipeline_stage
+                    - some_other_key_maybe_airflow_run_id
 
-            .. code-block:: python
+          Example Runtime Batch Request using an in-memory DataFrame:
 
-                from great_expectations.core.batch import BatchRequest
-
-                batch_request = BatchRequest(
-                    datasource_name="insert_your_pandas_datasource_name_here",
-                    data_connector_name="insert_your_runtime_data_connector_name_here",
-                    data_asset_name="insert_your_data_asset_name_here",
-                    batch_data=insert_your_dataframe_here,
-                    data_connector_query={
-                        "batch_identifiers": {
-                            "some_key_maybe_pipeline_stage": "ingestion step 1",
-                            "some_other_key_maybe_run_id": "run 18"
-                        }
+          .. code-block:: python
+
+            from great_expectations.core.batch import RuntimeBatchRequest
+            import pandas as pd
+
+            df: pd.DataFrame = pd.read_csv("some_path.csv")
+            runtime_batch_request = RuntimeBatchRequest(
+                datasource_name="my_pandas_datasource",
+                data_connector_name="my_runtime_data_connector",
+                data_asset_name="insert_your_data_asset_name_here",
+                runtime_parameters={
+                  "batch_data": df
+                },
+                batch_identifiers={
+                    "some_key_maybe_pipeline_stage": "ingestion step 1",
+                    "some_other_key_maybe_run_id": "run 18"
+                }
+            )
+
+          Example Runtime Batch Request using a path:
+
+          .. code-block:: python
+
+            from great_expectations.core.batch import RuntimeBatchRequest
+
+            path = "some_csv_file_with_no_file_extension"
+            runtime_batch_request = RuntimeBatchRequest(
+                datasource_name="my_pandas_datasource",
+                data_connector_name="my_runtime_data_connector",
+                data_asset_name="insert_your_data_asset_name_here",
+                runtime_parameters={
+                    "path": path
+                },
+                batch_identifiers={
+                    "some_key_maybe_pipeline_stage": "ingestion step 1",
+                    "some_other_key_maybe_run_id": "run 18"
+                },
+                batch_spec_passthrough={
+                    "reader_method": "read_csv",
+                    "reader_options": {
+                        "sep": ",",
+                        "header": 0
```

Comment:
I wasn't successful in creating a validator with this runtime_batch_request... (I received an error with the `reader_method`) Were you able to get it working on your end?

Reply:
Weird. It's working for me. What was the error you were getting?